src/Workspaces/Core/Desktop/Workspaces.Desktop.csproj

```diff
@@ -31,15 +31,6 @@
     <DefineConstants>$(DefineConstants);MSBUILD12</DefineConstants>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.Build, Version=$(VisualStudioReferenceAssemblyVersion), Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
-      <Private>false</Private>
-    </Reference>
-    <Reference Include="Microsoft.Build.Framework, Version=$(VisualStudioReferenceAssemblyVersion), Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
-      <Private>false</Private>
-    </Reference>
-    <Reference Include="Microsoft.Build.Tasks.$(MSBuildAssemblyNameFragment), Version=$(VisualStudioReferenceAssemblyVersion), Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
```

Comment:
Turns out [VSL.Imports.Targets](https://github.com/dotnet/roslyn/blob/master/build/Targets/VSL.Imports.targets#L121) uses this variable, and few closed side projects set `GenerateMSBuildBindingRedirects` to true. We can get rid of this variable once `GenerateMSBuildBindingRedirects` is removed - I'll do this after my closed side change.